File: server/src/main/java/org/babyfish/jimmer/client/generator/openapi/OpenApiProperties.java
package org.babyfish.jimmer.client.generator.openapi;


/**
 * 移除jimmer.client后配置报错，用来占位的
 */
public class OpenApiProperties {
    public static class Info{}


    public static Builder newBuilder(OpenApiProperties object) {
        return new Builder();
    }

     public static InfoBuilder newInfoBuilder(Info o) {
        return new InfoBuilder();
    }

    public static class Builder {


        public Builder setInfo(Info info) {
            return this;
        }



        public OpenApiProperties build() {
            return null;
        }
    }

    public static class InfoBuilder {

        public InfoBuilder setTitle(String title) {

            return this;
        }

        public InfoBuilder setDescription(String description) {

            return this;
        }


        public InfoBuilder setVersion(String version) {
            return this;
        }

        public Info build() {
            return null;
        }
    }

}

File: shared/src/jvmMain/kotlin/org/babyfish/jimmer/client/generator/openapi/OpenApiProperties.kt
package org.babyfish.jimmer.client.generator.openapi


/**
 * 移除jimmer.client后配置报错，用来占位的
 */
class OpenApiProperties {
    companion object{

        @JvmStatic
        fun newBuilder(o: OpenApiProperties?): Builder {
            return Builder()
        }
        @JvmStatic
        fun newInfoBuilder(o: Info?): InfoBuilder {
            return InfoBuilder()
        }
    }

    class Info


    class Builder {
        fun setInfo(info: Info?): Builder {
            return this
        }


        fun build(): OpenApiProperties? {
            return null
        }
    }

    class InfoBuilder {
        fun setTitle(title: String?): InfoBuilder {
            return this
        }

        fun setDescription(description: String?): InfoBuilder {
            return this
        }


        fun setVersion(version: String?): InfoBuilder {
            return this
        }

        fun build(): Info? {
            return null
        }
    }
}